examples/langchain-llama2-chatbot-sagemaker/README.md

```diff
@@ -21,14 +21,21 @@ As illustrated above, our example application relies on the following AWS servic
 
 The table below shows a comparison between Pluto and other tools regarding the amount and type of code:
 
-|             | Code Type                          | Lines of Code |
-| ----------- | ---------------------------------- | ------------- |
-| Application | Business Logic                     | 79            |
-| Pulumi      | Infrastructure Configuration       | 159           |
-| Terraform   | Infrastructure Configuration       | 201           |
-| **Pluto**   | **Business Logic + Configuration** | **87**        |
-
-Using Pluto, you can integrate business logic and infrastructure configuration with only an 8-line increase compared to pure business logic code, significantly reducing the amount of code. When building with Pulumi or Terraform, the total lines of code for business logic plus infrastructure configuration reach 238 and 280 lines, respectively, with Pluto's code amounting to only about a third of that.
+|                                                         | Code Type                          | Lines of Code        | Code Location                    |
+| ------------------------------------------------------- | ---------------------------------- | -------------------- | -------------------------------- |
+| Application                                             | Business Logic                     | 83(.ts)              | [Code](./comparison/application) |
+| [Terraform](https://www.terraform.io/)                  | Infrastructure Configuration       | 201(.tf)             | [Code](./comparison/terraform)   |
+| [Pulumi](https://www.pulumi.com/)                       | Infrastructure Configuration       | 157(.ts)             | [Code](./comparison/pulumi)      |
+| [Pulumi Serverless](https://www.pulumi.com/serverless/) | Business Logic + Configuration     | 256(.ts)             | [Code](./comparison/pulumi-app)  |
+| [Winglang - TS](https://github.com/winglang/wing)       | Business Logic + Configuration     | 100(.ts)             | [Code](./comparison/wing-ts)     |
+| [Winglang - Wing](https://github.com/winglang/wing)     | Business Logic + Configuration     | 71(.w) + 47(.ts)     | [Code](./comparison/wing-wing)   |
+| [Lepton](https://www.lepton.ai/)                        | Business Logic + Configuration     | 116(.py)             | [Code](./comparison/lepton)      |
+| [Serverless](https://github.com/serverless/serverless)  | Business Logic + Configuration     | 79(.ts) + 120(.yaml) | [Code](./comparison/serverless)  |
+| **Pluto**                                               | **Business Logic + Configuration** | **100(.ts)**         | [Code](./src)                    |
```

Comment:
How many lines is the Python code approximately?

Reply:
If it's Pluto, around 72 lines, not sure about other tools.

Reply:
> If it's Pluto, around 72 lines, not sure about other tools.

You can write these information in this document.